### DISCUSSION TITLE
do you know when the Dev channel will include September WSL needed update?

### DISCUSSION THREAD

**Discussion by iosub** [microsoft/WSL#10598] [Q&A]
hi, do you know when the Dev channel will include September WSL needed update? Yesterday Build 23560 does not include yet. Thanks

**Comment by nlvw** [comment]
Win 11 Insider Release preview 23H2 22631.2361 has everything.  At Least it works for me.

**Comment by iosub** [comment]
> Win 11 Insider Release preview 23H2 22631.2361 has everything. At Least it works for me.


Yes I Know, but my question was about Dev channel

Thanks
